lapis/src/test/kotlin/org/genspectrum/lapis/silo/SiloQueryTest.kt

```diff
@@ -62,6 +64,89 @@ class SiloQueryTest {
         assertThat(objectMapper.readTree(result), equalTo(objectMapper.readTree(expected)))
     }
 
+    @Test
+    fun `GIVEN OrderBySpec Random without seed THEN serializes with randomize true`() {
```

Comment:
Is there a reason why you didn't put those tests into the `MethodSource` of the `SiloAction is correctly serialized to JSON` test above?

Reply:
no reason. Fixed it.